doc/release-notes/0.36/0.36.md

```diff
@@ -48,18 +48,46 @@ The following table covers notable changes in v0.36.0. Further information about
 </thead>
 <tbody>
 
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/16256">#16256</a></td>
+<td valign="top">The location of the default directory for the shared cache and snapshot is changed to the <tt>.cache/javasharedresources</tt> in the user's home directory.</td>
+<td valign="top">All versions (AIX&reg;, Linux&reg;, macOS&reg;)</td>
+<td valign="top">This change is for non-Windows&trade; and non-z/OS&reg; platforms and applies only if <tt>-Xshareclasses:groupAccess</tt> is not used. If <tt>groupAccess</tt> is used, the directory that gets set remains as <tt>/tmp/javasharedresources/</tt>.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/16133">#16133</a></td>
+<td valign="top">New <tt>-XX:[+|-]MergeCompilerOptions</tt> option is added to enable or disable the merging of multiple -Xjit or -Xaot options into a single -Xjit or -Xaot option.</td>
+<td valign="top">All versions</td>
+<td valign="top"> By default, if you specify <tt>-Xjit</tt> option (or <tt>-Xaot</tt> option) multiple times, only the last option takes effect. Therefore, on adding an <tt>-Xjit</tt> option, the resultant behavior might not be as expected, if any instance of this option already exists. You can use the <tt>-XX:+MergeCompilerOptions</tt> option to merge all the existing instances of the <tt>-Xjit</tt> options logically.</td>
+</tr>
+
+<tr>
+<td valign="top"><a href="https://github.com/eclipse-openj9/openj9/issues/16166">#16166</a></td>
+<td valign="top">Default JITServer AOT cache is named as <tt>default</tt>.</td>
+<td valign="top">All versions (Linux)</td>
+<td valign="top">A JITServer instance can have several AOT caches, each with its own name. Client JVMs with different profiles of execution can specify a particular AOT cache name with the <tt>-XX:JITServerAOTCacheName</tt> option to use the same JITServer instance. Earlier, if the cache name was not specified in this option, the default was to use a nameless cache. The default AOT cache name is now changed to <tt>default</tt>.</td>
```

Comment:
Maybe I am picky, but reading again this sentence:
"Client JVMs with different profiles of execution can specify a particular AOT cache name with the <tt>-XX:JITServerAOTCacheName</tt> option to use the same JITServer instance."
it makes me think that it could be misinterpreted as "if you want two clients to connect to the same JITServer instance you must use `XX:JITServerAOTCacheName`"
Enforcing two clients to connect to the same JITServer instance is done by specifying the same JITServer address.
The paragraph should convey that, if two clients with different profiles of execution connect to the same JITServer instance, they should target different AOT caches with the `XX:JITServerAOTCacheName` option, caches that contain methods optimized based on their own execution profile.

A suggestion for rephrasing:
<td valign="top">A JITServer instance can have several AOT caches, each with its own name. The <tt>-XX:JITServerAOTCacheName</tt> option allows client JVMs with different profiles the execution to connect to the same JITServer instance, but access different versions of compiled methods optimized for their own profile. Earlier, if the cache name was not specified in this option, the default was to use a nameless cache. The default AOT cache name is now changed to <tt>default</tt>.</td>